packages/blocks/blocksAntd/src/blocks/Alert/Alert.js

```diff
@@ -37,8 +37,11 @@ const AlertBlock = ({ blockId, events, methods, properties }) => {
       description={renderHtml({ html: properties.description, methods })}
       id={blockId}
       message={
-        renderHtml({ html: properties.message, methods }) ||
-        (type.isNone(properties.description) && <div style={{ height: '1.5175em' }}></div>)
+        type.isNone(properties.message) ? (
```

Comment:
Maybe we could a `default` option to the `renderHtml ` function?

Reply:
Think this is ok for now, this seems like an edge case with alert.